test/fsm_eqc_vnode.erl

```diff
@@ -99,9 +98,14 @@ get_postcommits() ->
 init([]) ->
     {ok, active, #state{}}.
 
-active(?VNODE_REQ{index=Idx,
-                  sender=Sender,
-                  request=?KV_GET_REQ{req_id=ReqId}}, State) ->
+active(?VNODE_REQ{index = Idx,
```

Comment:
The `VNODE_REQ` record is defined in `riak_core_vnode.hrl` and since our scope is just the request records defined in `riak_kv_vnode.hrl` we left this one as-is. Per @lehoff this is for another round.